database/01_table_creation.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE token_info (
+    token_address bytea PRIMARY KEY,
+    symbol varchar NOT NULL,
+    decimals int NOT NULL
+);
+
+CREATE TABLE token_times (
```

Comment:
I find the name a bit unintuitive. I would probably change it to something like

`transaction_timestamps`

Reply:
Would `token_timestamps` work? Or `token_transactions`?

This table is not about imbalances and it is not about transactions. The table might be superseded by a `token_imbalances` table for linking tokens to transactions, maybe in combination with a `settlements` table linking transaction hashes and times.

Reply:
> This table is not about imbalances and it is not about transactions.
Hm, then what is it about? There is a tx hash associated with each entry

Reply:
I added a clarification in the comment above.

* For a table with the name `transaction_timestamps` i would expect that `(tx_hash, time)` is a key. Having token addresses in that would surprise me.
* For a table with the name `imbalance_timestamps` i would expect that it stores imbalances in some form.

Maybe the clean design would be to have a `transaction_timestamps` table and a `transaction_tokens` table. To avoid overhead due to additional tables it could also be a `transaction_token_timestamps` table with columns `(tx_hash, token_address, time)` or something like that.

Reply:
Yeah, the distinction between the `transaction_timestamps` and `transaction_tokens` tables make sense. And actually we could easily generate the `transaction_tokens` table already by reducing the scope of the part of the code that computes imbalances (it could simply just keep track of what tokens are being transferred and record those in this table). I.e., repurpose the `raw_token_imbalances` table into a `transaction_tokens` table until further notice.